components/raftstore/src/store/peer.rs

```diff
@@ -1519,20 +1489,12 @@ where
             "peer_id" => self.peer.get_id(),
         );
 
-        let mut ready = self.raft_group.ready_since(self.last_applying_idx);
+        let mut ready = self.raft_group.ready();
```

Comment:
Does `last_applying_idx` is not necessary? Would the method `ready()` record and update last applying index?

Reply:
Yes. As you said, the raft-rs records and updates the index of the last committed entries given before.

Reply:
I will add an assert to make sure they are the same.

Reply:
Oh. I think the assert in `handle_raft_committed_entries` in apply fsm is enough for this case.🤣